lib/remote/services/comment_service.chopper.dart

```diff
@@ -0,0 +1,53 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
again out of curiosity so that I start learning more about this code :)
What is this chopper for? Is this supposed to come in git?

Reply:
@isabelcosta this is just built by [build_runner](https://pub.dev/packages/build_runner) . We can add this to gitignore, but if the testers wanted to test the app they would have to genrate these files on thier PC. This would affect the testing so we just decided to keep this aa it is

Reply:
More info about this [here](https://github.com/anitab-org/mentorship-flutter/pull/103#issuecomment-647046746)